lib/camp.js

```diff
@@ -712,6 +712,10 @@ function Path(matcher) {
       return path.match(matcher);
     };
   } else if (typeof matcher === 'string') {
+    if (matcher[0] !== '/') {
+      matcher = '/' + matcher;
+      this.matcher = matcher;
```

Comment:
Probably cleaner to define `this.matcher` once and for all (remove the `this.matcher =` at the top) at the end of this function's body instead?